src/lib.rs

```diff
@@ -93,30 +93,11 @@
 //! * If you have experience with NumPy, you may also be interested in
 //!   [`ndarray_for_numpy_users`](doc/ndarray_for_numpy_users/index.html).
 
-#[cfg(feature = "serde-1")]
-extern crate serde;
-
-#[cfg(feature = "rayon")]
-extern crate rayon;
-
-#[cfg(feature = "approx")]
-extern crate approx;
-
 #[cfg(feature = "blas")]
 extern crate blas_src;
```

Comment:
Not sure why, but these blas `extern crate`s seem to be required...?

Reply:
Huh, based on the error messages, it seems to have something to do with linking. The blas crates link to non-Rust code, so I assume that's what makes them different. Leaving in the `extern crates` is fine since they seem to be necessary to compile successfully.